3-UsingAPIs/Week2/assignment/ex4-diceRace.js

```diff
@@ -15,20 +15,25 @@ import { rollDie } from '../../helpers/pokerDiceRoller.js';
 
 export function rollDice() {
   const dice = [1, 2, 3, 4, 5];
-  // TODO complete this function; use Promise.race() and rollDie()
-  rollDie(1); // TODO placeholder: modify as appropriate
+  const allPromises =  dice.map((element) => rollDie(element));
```

Comment:
This can be shorted to:

```suggestion
  const allPromises =  dice.map(rollDie);
```